CMakeLists.txt

```diff
@@ -217,9 +217,9 @@ KOKKOSKERNELS_ADD_OPTION(
 )
 
 SET(EXECSPACE_CUDA_VALID_MEM_SPACES      CUDASPACE CUDAUVMSPACE)
-SET(EXECSPACE_SERIAL_VALID_MEM_SPACES    HBWSPACE HOSTSPACE CUDAUVMSPACE)
-SET(EXECSPACE_OPENMP_VALID_MEM_SPACES    HBWSPACE HOSTSPACE CUDAUVMSPACE)
-SET(EXECSPACE_PTHREAD_VALID_MEM_SPACES   HBWSPACE HOSTSPACE CUDAUVMSPACE)
+SET(EXECSPACE_SERIAL_VALID_MEM_SPACES    HBWSPACE HOSTSPACE)
```

Comment:
@jjwilke do these changes for valid mem spaces need to propagate to develop independent of this PR?

Reply:
No, this is only an ETI thing.

Reply:
so we're double-instantiating all those kernels?  sadness :(

Reply:
I think we were before, too? This PR is a "preserve all existing behavior". The next PR would do the anon spaces stuff.

One thing at a time!

Reply:
@jjwilke praise Turing and pass the ammunition ;-)